WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -1260,8 +1260,7 @@ fileprivate extension AztecPostViewController {
     }
 
     func markBlogAsLastUsed(_ blog: Blog) {
```

Comment:
Should we just ditch this method completely and call the `RecentSitesService` directly?